docs/selectize.html

```diff
@@ -4,9 +4,9 @@
     <meta charset="utf-8" />
     <title>JSON Editor Selectize Integration Example</title>
     <script src="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/jsoneditor.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/selectize.js/0.12.1/js/standalone/selectize.js"></script>
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/selectize.js/0.12.1/css/selectize.bootstrap3.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
```

Comment:
Was there a special reason for changing this?

Reply:
I couldn't get it to work with the original libs.

Reply:
I meant using `https://` instead of `//`. That was the only change.